modular_skyrat/modules/rod-stopper/code/immovable_skyrat.dm

```diff
@@ -0,0 +1,9 @@
+/obj/effect/immovablerod/Bump(atom/clong)
+	var/should_self_destroy = FALSE
+	if(istype(clong, /obj/machinery/rodstopper))
+		should_self_destroy = TRUE
+	. = ..()
+	if(should_self_destroy)
```

Comment:
maybe make one of those BOH explosions when it happens

Reply:
Infact yes, do that instead

Reply:
That's a whooole lot of station destroyed, but I'll look into it.

Reply:
Welp, it's in.